README.md

```diff
@@ -204,7 +208,7 @@ For the uninitiated, MAX_PATH is a limitation with many Windows tools and APIs t
   moves duplicate packages to top-level
 * `> npm install -g flatten-packages`
   moves all packages to top-level, but can cause versioning issues
-* :heart: Upgrade to npm@3
+* :heart: Upgrade to npm@3 which flattens out dependencies and so avoids nested node_module folders.
```

Comment:
let's be more precise on this - npm@3 makes things *maximally flat* - you can still run into issues when versions conflict.